### DISCUSSION TITLE
Request for adding AVRDUDE to Windows Package Manager (Winget)

### DISCUSSION THREAD

**Discussion by hayschan** [avrdudes/avrdude#1448] [Ideas]
Hello AVRDUDE team,

I would like to request the addition of AVRDUDE to the Windows Package Manager (Winget). As a Linux user, I have enjoyed the convenience of being able to install AVRDUDE with a single command using the package manager (e.g. `sudo apt-get install avrdude`). However, I have noticed that Windows users currently do not have this option, and instead must manually download and install AVRDUDE, which can be inconvenient and time-consuming.

AVRDUDE is a widely used utility for programming AVR microcontrollers, and it is available on many platforms including Windows. By making AVRDUDE available through Winget, Windows users would be able to install and update AVRDUDE with a simple command `winget install avrdude`, and benefit from the convenience and reliability of a package manager.

I believe that adding AVRDUDE to Winget would greatly improve the user experience for Windows users who rely on this utility for their projects. It would also help to promote the use of AVRDUDE among Windows users, and encourage more people to use AVR microcontrollers for their projects.

Thank you for your consideration, and please let me know if there is anything I can do to support this request.

Best regards,
Hays Chan

**Comment by mcuee** [comment]
You can request through winget?
https://learn.microsoft.com/en-us/windows/package-manager/package/

**Reply by hayschan** [reply]
Done. Posted a Winget Package Request of avrdude. https://github.com/microsoft/winget-pkgs/issues/110758

**Comment by mcuee** [comment]
FYI scoop comes with avrdude 7.1 as of now.
https://scoop.sh/#/apps?q=avrdude
https://github.com/ScoopInstaller/Main/blob/master/bucket/avrdude.json

**Comment by mcuee** [comment]
Unfortunately choco still comes with older avrdude 6.3 version.
https://community.chocolatey.org/packages/avrdude

**Comment by mcuee** [comment]
https://github.com/microsoft/winget-pkgs/issues/110758 has been resolved by Microsoft.
